internal/service/image.go

```diff
@@ -5,23 +5,17 @@ import (
 	"context"
 	"fmt"
 	"net/http"
-	"strconv"
 
 	"github.com/kubev2v/migration-planner/internal/api/server"
 	"github.com/kubev2v/migration-planner/internal/image"
 )
 
 func (h *ServiceHandler) GetSourceImage(ctx context.Context, request server.GetSourceImageRequestObject) (server.GetSourceImageResponseObject, error) {
-	id, err := strconv.ParseUint(request.Id, 10, 32)
-	if err != nil {
-		return server.GetSourceImage400JSONResponse{Message: "invalid ID"}, nil
-	}
-
 	writer, ok := ctx.Value(image.ResponseWriterKey).(http.ResponseWriter)
 	if !ok {
 		return server.GetSourceImage400JSONResponse{Message: "error creating the HTTP stream"}, nil
 	}
-	ova := &image.Ova{Id: id, Writer: writer}
+	ova := &image.Ova{Id: request.Id, Writer: writer}
```

Comment:
@avishayt is there any chance we can add the `content-disposition` header to the response (GetSourceImage200ApplicationoctetStreamResponse)? That will allow the UI to trigger the file download